addons/mail/models/mail_thread.py

```diff
@@ -2550,7 +2552,7 @@ def _notify_get_groups(self):
             )
         ]
 
-    def _notify_classify_recipients(self, recipient_data, model_name):
+    def _notify_classify_recipients(self, recipient_data, model_name, access_link):
```

Comment:
As we target a stable we cannot add a new mandatory parameter. This would make override crash.

What we could do is to add an optional msg_vals parameter to _notify_classify_recipients and give it to _notify_get_action_link, like

```
def _notify_classify_recipients(self, recipient_data, model_name, msg_vals=None):
    ...
    access_link = self._notify_get_action_link('view', **msg_vals)
    ...
```
That way diff is minimal. Would that solve your issue ?

cc @Xavier-Do